src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
-
 import CabinList from './components/CabinList';
+import cabinData from './data/cabin';
 
 function App() {
-  return <CabinList />;
+  const cabinNameList1 = cabinData.map((data) => data.cabin);
+  const cabinNames = [...new Set(cabinNameList1)];
+  // const cabin = cabinNames.map((data, index),
+
+  return <CabinList name={cabinNames} />;
```

Comment:
i would probably change the name of the prop here to be something more related to what the data is.  E.g cabinData or cabins.